kubectl-rook

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The Rook Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
```

Comment:
I usually do `set -eEuo pipefail` for all my scripts.

-E allows subshells to inherit the exit status in some bash versions.
-u ensures that there are no unset vars
- pipefail ensures that any piped commands fail properly

Reply:
I didn't add `o pipefail` as I used my `|` (at least in my mind) but yeah good to have

Reply:
and I had to use `/bin/bash` as entry point because it was giving an error...let me know if you have other suggestions